sprokit/src/sprokit/python/util/python_exceptions.h

```diff
@@ -38,42 +38,48 @@ namespace python {
 
 /// \todo More useful output?
 
-#define SPROKIT_PYTHON_HANDLE_EXCEPTION(call)     \
-  try                                             \
-  {                                               \
-    call;                                         \
-  }                                               \
-  catch (pybind11::error_already_set const&) \
-  {                                               \
-    sprokit::python::python_print_exception();    \
-                                                  \
-    throw;                                        \
+#define SPROKIT_PYTHON_HANDLE_EXCEPTION(call)                     \
+  try                                                             \
+  {                                                               \
+    call;                                                         \
+  }                                                               \
+  catch (pybind11::error_already_set const& e)                    \
+  {                                                               \
+    auto logger = kwiver::vital::get_logger("python_exceptions"); \
+    LOG_WARN(logger, "Handle Python Exception:");                 \
+    LOG_WARN(logger, e.what());                                   \
```

Comment:
This could be all on one line:

  `LOG_WARN(logger, "Handle Python Exception: " << e.what() );`

Reply:
will fix.